src/algebra/group/conj.lean

```diff
@@ -45,8 +69,136 @@ calc is_conj a 1 ↔ is_conj 1 a : ⟨is_conj_symm, is_conj_symm⟩
 lemma conj_injective {x : α} : function.injective (λ (g : α), x * g * x⁻¹) :=
 (mul_aut.conj x).injective
 
-@[simp] lemma is_conj_iff_eq {α : Type*} [comm_group α] {a b : α} : is_conj a b ↔ a = b :=
-⟨λ ⟨c, hc⟩, by rw [← hc, mul_right_comm, mul_inv_self, one_mul], λ h, by rw h⟩
+end group
 
-protected lemma monoid_hom.map_is_conj (f : α →* β) {a b : α} : is_conj a b → is_conj (f a) (f b)
-| ⟨c, hc⟩ := ⟨f c, by rw [← f.map_mul, ← f.map_inv, ← f.map_mul, hc]⟩
+@[simp] lemma group_with_zero.is_conj_iff [group_with_zero α] {a b : α} :
```

Comment:
Elsewhere we just use a primed version for the `group_with_zero` version of `group` lemmas - you could likely do the same here to keep the names short.